contrib/ci/check-quirks.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2021 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1+
+
+import glob
+import os
+import sys
+
+
+def test_files() -> int:
+
+    rc: int = 0
+
+    for fn in glob.glob("**/*.quirk", recursive=True):
+        with open(fn, "r") as f:
+            for line in f.read().split("\n"):
+                if line.startswith(" ") or line.endswith(" "):
```

Comment:
can we detect the deprecated `DeviceInstanceId` prefix too?

Reply:
Yup, good idea. And Guid.